### ISSUE TITLE
comment syntax

### ISSUE THREAD

**Issue by loganwright** [vapor/leaf-kit#15]
Original comment syntax of:

```
        #("foo")
        #// this is a comment!
        bar
```

and multi-line of:

```
        #("foo")
        #/*
            this is a comment!
        */
        bar
```

not currently supported, and conflicts with our new preference of `:` to open a body and subsequent `#end<tag>` syntax.

I propose moving to:

```
#com:
  this is a comment here :)
#endcom
```

**Comment by tdotclare** [comment]
I'd really like to introduce commenting for the new syntax, and personally think the solution for readability and function is as simple as this. Tag indicators are not needed inside parameter lists and the observable expectation that an anonymous tag is a passthrough mechanism makes this both easy to parse and easy to read without adding additional characters, complicated rules, or lengthy body closures.
```
#(# Single Line Comment #)
#(#
Multi Line Comment
#)

**Comment by tdotclare** [comment]
Closed in LeafKit-1.0.0tau
